packages/bbob-plugin-helper/src/TagNode.ts

```diff
@@ -76,6 +78,14 @@ export class TagNode<TagValue extends any = any> implements TagNodeObject {
     return appendToNode(this, value);
   }
 
+  setStartTagPos(start: number, end: number) {
```

Comment:
I use one-word setters here. Look at `attr`

`start(from: number, to: number)` looks better here

Reply:
Same for `setEndTag`